S/Stride/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+using BinaryBuilder, Pkg
+using BinaryBuilderBase: Dependency
+
+name = "Stride"
+version = v"1.0.0"
+
+# url = "https://github.com/MDAnalysis/stride"
+# description = "Protein secondary structure assignment from atomic coordinates"
+
+sources = [
+    GitSource("https://github.com/MDAnalysis/stride",
+              "867a5eb0f2479cb16615512a53ee472c54649505"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/stride*/
+
+cd src
+
+make CC="${CC} -g -O2 -fPIC -Wall"
+mkdir -p "${bindir}"
+install -Dvm 755 "stride${exeext}" "${bindir}/"
```

Comment:
https://discourse.julialang.org/t/binarybuilder-error-loaderror-cannot-continue-with-unsatisfied-build-products/100992/13
```suggestion
install -Dvm 755 "stride" "${bindir}/strid${exeext}"
```